### PR TITLE
Fix slow sub desc

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

**Review by tigercl — APPROVED** [review]

File: en_US/advanced/alarms.md [hunk 1]
```diff
@@ -52,3 +52,11 @@ EMQX Broker allows users to adjust the alarm function to a certain extent to mee
 | alarm.validity_period         | duration | 24h           | The maximum storage time of deactivated alarms, and expired alarms will be cleared |
 
 
+{% emqxee %}
+EMQX Enterprise raises an alarm when the license expiration date is less than 30 days or if the number of connections exceeds the high water mark. The user can adjust the high/low water mark for the number of connections according to the actual situation.
```
Comment (k32) [inline en_US/advanced/alarms.md:56]:
```suggestion
EMQX Enterprise raises an alarm when the license expirates in less than 30 days, or if the number of connections exceeds the high watermark. User can adjust the high/low watermark for the number of connections according to the actual situation.
```

File: en_US/configuration/configuration.md [hunk 2]
```diff
@@ -7732,3 +7732,44 @@ Forward the `on_message_delivered` event.
...
+<br />
+### license.connection_high_watermark_alarm
+
+| Type     | Default                              |
+| -------- | ------------------------------------ |
+| percent   | 80% |
+
+#### Description
+
+The alarm is raised when this threshld is reached. As percentage of alive connections/max connections.
```
Comment (k32) [inline en_US/configuration/configuration.md:7757]:
```suggestion
The alarm is raised when this threshold is reached. As percentage of alive connections/max connections.
```

File: en_US/configuration/configuration.md [hunk 3]
```diff
@@ -7732,3 +7732,44 @@ Forward the `on_message_delivered` event.
...
+- After the alarm occurs, you can refer to [How to update the license?] (../faq/use-guide.md#how-to-update-emq-x-license) for hot update.
+- When the number of connections exceeds the maximum allowed value, new client connections will be rejected, and already connected clients will not be affected.
```
Comment (k32) [inline en_US/configuration/configuration.md:7759]:
```suggestion
- When the number of connections exceeds the maximum allowed value, new client connections will be rejected, but the already connected clients will not be affected.
```

File: en_US/modules/slow_subscribers_statistics.md [hunk 4]
```diff
@@ -18,7 +18,9 @@ Then, select the **Slow Subscribers Statistics** module, and then click *Enable*
 
 ## Implementation note
 
-With this feature enabled, EMQ X will start measuring message transmission latency. The measurement always starts at when a message is received by EMQ X, the end of the measurement is configurable. See more details...
+With this feature enabled, EMQX will start measuring message transmission latency.
+The measurement always starts at when a message is received by EMQX,
```
Comment (k32) [inline en_US/modules/slow_subscribers_statistics.md:22]: "at when" — duplicate words?

**Pull request merged**